### DISCUSSION TITLE
How many lines of code are generate by expanding #[automock]?

### DISCUSSION THREAD

**Discussion by becothas** [asomers/mockall#562] [Questions] [answered]
Hi. 
I am mocking(`#[automock]`) a module with 89 `extern"C"` functions generated by bindgen. And it generates around 70k lines of code for it, which makes my rust-analyzer very laggy. 

Is it expected to generate so much code?

**Comment by asomers** [comment, marked as answer]
Yes, that sounds about right.  If your editor is too laggy, you could configure it to analyze in non-test mode.

**Reply by becothas** [reply]
Wow.. That's a lot.
I will try it. 
Thanks.
